docs/developers_notes/02-base_regressor.md

```diff
@@ -59,7 +59,16 @@ When devising a new model subclass based on the `BaseRegressor` abstract class,
 - **Must** realize the abstract methods, see above.
 - **Should not** overwrite the `get_params` and `set_params` methods, inherited from `Base`.
 - **May** introduce auxiliary methods for added utility.
+- **May** add re-implement the `__sklearn_tags__` method adding metadata that is relevant to the specific estimator implemented. See the [`scikit-learn` Tags](https://scikit-learn.org/stable/modules/generated/sklearn.utils.Tags.html#sklearn.utils.Tags) for the available tagging options.
```

Comment:
```suggestion
- **May** re-implement the `__sklearn_tags__` method to add metadata that is relevant to the specific estimator implemented. See the [`scikit-learn` documentation](https://scikit-learn.org/stable/modules/generated/sklearn.utils.Tags.html#sklearn.utils.Tags) for the available tagging options.
```
I think I'd add [this page](https://scikit-learn.org/stable/developers/develop.html#estimator-tags) here as well (not just in the admonition).

Reply:
maybe worth giving an example here? right now, it's not clear to me when I'd write additional tags.